tests/test_state_props/test_schmidt_rank.py

```diff
@@ -6,7 +6,7 @@
 from toqito.states import basis, bell
 
 
-e_0, e_1 = np.array([[1], [0]]), np.array([[0], [1]])
+bas_0, bas_1 = np.array([[1], [0]]), np.array([[0], [1]])
```

Comment:
We usually use `e_0` and `e_1` elsewhere in the code to denote these standard basis vectors. Any reason to change that convention here? I'm fine with either, just curious.

Reply:
Pylint flagged them. So, I changed them.

Reply:
Huh, interesting. Curious that it doesn't complain about this convention elsewhere. Is it maybe something that's only occurring locally? If not, I wonder if it's worthwhile to set something in the settings as I think I disagree with the linter and think that `e_0` and `e_1` are okay.

Reply:
>Is it maybe something that's only occurring locally?

Nope. https://github.com/vprusso/toqito/actions/runs/6872626793/job/18691377546#step:6:14

>If not, I wonder if it's worthwhile to set something in the settings 

Not sure. I'll check.

Reply:
Ah, looking at the error, yeah I think I know why. I think we just want the definition of `e_0` and `e_1` at the top of the file (and then remove the definitions of `e_0` and `e_1` elsewhere in the file. That should fix this issue!

Reply:
Stange. My first attempt at fixing this was doing what you just said I try. At the time, tests were failing. But now, they work. :shrug:

Reply:
If I had a nickel for every time something like that happened to me, I'd be a wealthy man 💰 

Glad it works now!